README.md

```diff
@@ -0,0 +1,114 @@
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+Hello-microverse is a repository created for educational purposes, to show tools and best practices a developer must master when creating files, branches and changes in a project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+VS Code editor
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML and CSS
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[HTML file]**
+- **[CSS file]**
+- **[Linters installed: Lighthouse, Webhint, Stylelint and node_modules]**
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+NOT AVAILABLE
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This project shows tools and best practices a developer must master when creating files, branches and changes in a project.
+
+### Prerequisites
+
+In order to run this project you need:
+
+VS Code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/eamador1/hello-microverse.git
+
+### Install
+
+Install this project with:
+
+VS Code, downloading repo.
+
+### Usage
+
+To run the project, execute the following command:
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+npm install --save-dev hint@7.x
+npx hint .
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Deployment
+
+You can deploy this project using: Web host provider
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Erwin E Amador
```

Comment:
[OPTIONAL] Very good job with the **_Authors_** section 🥇 I think it would be better if you provide your social media profiles links in the Author's section as well 🔗 This is a great way to build a network 💪 What do you think about it ? 💭 For example, you can use something like below 👇


- GitHub: [your GitHub handle]()
- Twitter: [your Twitter handle]()
- LinkedIn: [your LinkedIn handle]()